zproject/default_settings.py

```diff
@@ -47,6 +47,11 @@
 
 # Social auth; we support providing values for some of these
 # settings in zulip-secrets.conf instead of settings.py in development.
+SOCIAL_AUTH_APPLE_ID_CLIENT = get_secret('social_auth_apple_id_client')
```

Comment:
What's the reason for using `get_secret` for  the id_client?

Reply:
I'm thinking we'll have id_client in `dev_secrets.conf` or `zulip_secrets.conf`.

Reply:
(This is another example of where it's confusing that there's an extra "id" here before "client" :smile: -- the name is trying to say "social auth / Apple ID / client", but as you can see, it's very natural to read it instead as "social auth / Apple / id_client". Even though "id_client" is a bit of a mysterious name in itself.)

Reply:
I think using `get_secret` should be right here, because during development to this when we want to set this value to something, it becomes an obstacle while pushing to github. In production we ask the user to set this value in `prod_settings_template.py`.